client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -416,7 +416,7 @@ export const SeoForm = React.createClass( {
 		const activateVerificationServices = () => this.props.activateModule( siteId, 'verification-tools' );
 		const isSitePrivate = siteSettings && parseInt( siteSettings.blog_public, 10 ) !== 1;
 		const isJetpackUnsupported = siteIsJetpack && ! jetpackVersionSupportsSeo;
-		const isDisabled = isSitePrivate || isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
+		const isDisabled = isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
```

Comment:
Note: this will also enable the fields for when the site is `hidden`, because `isSitePrivate` is `true` for both hidden and private sites. Is that on purpose? I'm asking because it differs from what you're suggesting for the SEO description in editor drawer - it'll be hidden for `hidden` sites, but displayed for `private` sites. This is particularly important, considering the fact that `private` is not available for Jetpack sites, only `hidden`.